app/components/Utility/CryptoLinkFormatter.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import QRCode from "qrcode.react";
+
+/**
+ *  Given an asset name, amount and label generates uri for its refilling
+ *
+ */
+
+class CryptoLinkFormatter extends React.Component {
+    static assetTemplates = {};
+
+    static propTypes = {
+        asset: PropTypes.string,
+        address: PropTypes.string,
+        amount: PropTypes.number,
+        message: PropTypes.string,
+        size: PropTypes.number
+    };
+
+    static defaultProps = {
+        size: 140
+    };
+
+    constructor(params) {
+        super(params);
+
+        this.assetTemplates = {
```

Comment:
Please document how assetTemplates dictionary needs to look like, including bind and value usage

Reply:
stressed this info in comments